app/utils.py

```diff
@@ -58,7 +58,8 @@ async def redis_get(key: str) -> str:
     v = await redis_plugin.redis.get(key)
 
     if not v:
-        logging.warning(f"Key '{key}' not found in Redis DB.")
+        if not "tor_web_addr" in key:
```

Comment:
I'm thinking of whether t'd make sense to just set it to `logging.info` as this solution suppresses potentially useful information.

I think the way to go is log `tor_web_addr` problems as `logging.info` and everything else as `logging.warning` This should get rid of the spammy messages unless someone changes the default log level warning to info on purpose.